examples/panel.py

```diff
@@ -0,0 +1,30 @@
+"""Example to create a Panel of Ophyd Signals from an object"""
+import sys
+import numpy as np
+from ophyd import Device, Component as Cpt, Signal
+from ophyd.sim import SignalRO
+from qtpy.QtWidgets import QApplication
+import typhon
+
+
+class Sample(Device):
+    """Simulated Device"""
+    readback = Cpt(SignalRO, value=1)
+    setpoint = Cpt(Signal, value=2)
+    waveform = Cpt(SignalRO, value=np.random.randn(100, ))
+    image = Cpt(SignalRO, value=np.ones((100, 100)) * 455)
```

Comment:
```suggestion
    image = Cpt(SignalRO, value=np.random.randn((100, 100)) * 455)
```
This would make for a more interesting image, at least.